tools/ci/before_install.sh

```diff
@@ -11,6 +11,9 @@ else
   cp config/database.pg.yml config/database.yml
   psql -c "CREATE USER root SUPERUSER PASSWORD 'smartvm';" -U postgres
   export BUNDLE_WITHOUT=development
+
+  which bower || npm install -g bower
+  bower install --allow-root -F --config.analytics=false
```

Comment:
Wait, is before install run before each? I assumed before all, I'll fix this..  We *probably* need it for vmdb,  but possibly for not much else..